modules/nlu/src/backend/training-pipeline.ts

```diff
@@ -292,10 +307,12 @@ const TrainContextClassifier = async (
     return input.intents
       .filter(intent => intent.contexts.includes(ctx) && intent.name !== NONE_INTENT)
       .map(intent =>
-        intent.utterances.map(utt => ({
-          label: ctx,
-          coordinates: getCtxFeatures(utt, customEntities)
-        }))
+        intent.utterances
+          .filter(u => !u.augmented) // we don't want to train on augmented utterances as it would slow down training too much
+          .map(utt => ({
+            label: ctx,
+            coordinates: getCtxFeatures(utt, customEntities)
```

Comment:
complexentities are probably part of customEntities, you'll need to ignore them in ctxFeaturizer bc list and pattern are already in entity encoding